examples/pennylane_universal_classifier.py

```diff
@@ -0,0 +1,474 @@
+r"""
+.. _universal_classifier:
+
+Universal Quantum Classifier 
```

Comment:
Can we change the name to 'Single-qubit quantum classifier'? "universal" can confuse people, since lots of qclassifiers are computationally universal, but not as function approximators...

Reply:
Agreed. What about something "Data-reuploading classifer" to reflect the paper content?

Reply:
Done